install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -37,6 +37,15 @@ spec:
       rules:
         - expr: sum(os_image_url_override)
           record: os_image_url_override:sum
+    - name: mcc-drain-error
+      rules:
+        - alert: MCCDrainError
+          expr: |
+            mcd_drain_err > 0
```

Comment:
Shouldn't this be mcc_drain_err to match your change below?

Reply:
Yeah, I think you're right. Will fix!